doc/release/1.13.0-notes.rst

```diff
@@ -227,6 +251,8 @@ More ``linalg`` operations now accept empty vectors and matrices
 All of the following functions in ``np.linalg`` now work when given input
 arrays with a 0 in the last two dimensions: ``det``, ``slogdet``, ``pinv``,
 ``eigvals``, ``eigvalsh``, ``eig``, ``eigh``.
+arrays with a 0 in the last two dimensions: `det``, ``slogdet``, ``pinv``,
+``eigvals``, ``eigvalsh``, ``eig``, ``eigh``, ``cholesky``.
```

Comment:
OK, that should have fixed it up, when going back to the previous version of setting to zero, I got the pointer arithmetic wrong apparently.